CONTRIBUTING.md

```diff
@@ -66,7 +66,7 @@ This process builds RST documentation directly from the proto files, merges it w
 files, and then runs [Sphinx](http://www.sphinx-doc.org/en/stable/rest.html) over the entire tree to
 produce the final documentation. The following are some general guidelines around documentation.
 
-* Cross link as much as possible. Sphinx is fantastic at this. Use it! See ample examples with the
```

Comment:
Not sure this is a typo, I think the idea is to say that there are plentiful (ample) examples.

Reply:
Oh, that's right. Thanks for pointing it out.